src/main/java/org/fundacionjala/virtualassistant/controllers/mongo/RecordingController.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.virtualassistant.controllers.mongo;
+
+
+import org.fundacionjala.virtualassistant.models.mongo.Recording;
+import org.fundacionjala.virtualassistant.services.mongo.RecordingService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/recordings")
+public class RecordingController {
+
+    @Autowired
+    RecordingService recordingService;
+
+    @GetMapping()
+    public ResponseEntity<List<Recording>> getAllRecordings() {
+        List<Recording> recordings = recordingService.getAllRecordings();
```

Comment:
All this implementation is also in @SerranoAlejandroSelaya and @ivanMCJ implementation so please dont repleat this code and apply DRY